.gitignore

```diff
@@ -43,6 +44,9 @@ output/*/index.html
 # Sphinx
 docs/_build
 
+# VSCode 
```

Comment:
Don't adapt a project .gitignore to your environment.

You should use a global .gitignore for that: https://help.github.com/articles/ignoring-files/

Reply:
Oh, sorry, I wasn't paying attention. Now restored.